cli/command/trust/info.go

```diff
@@ -101,6 +101,38 @@ func lookupTrustInfo(cli command.Cli, remote string) error {
 	}
 	printSignatures(cli, remote, signatureRows)
 
+	roleWithSigs, err := notaryRepo.ListRoles()
+	if err != nil {
+		fmt.Fprintf(cli.Out(), "Error listing signature metadata for roles: %s\n", err)
+		return err
+	}
+
+	signerRoleToKeyIDs := make(map[string][]string)
+	baseRoleToKeyIDs := make(map[string][]string)
+
+	for _, roleWithSig := range roleWithSigs {
+		// TODO(ashfall) Can we get rid of this for loop? Perhaps find
```

Comment:
I don't think we have a notary function to filter delegation vs base roles at present.  I'm fine with the conditional logic living in this codebase for now